src/server.js

```diff
@@ -42,12 +45,15 @@ const listen = (port) => {
 };
 
 const port = process.env.PORT || 3002;
-if (port > 3002) {
-  listen(port);
-} else {
-  findPort(port, port + 100, function(ports) {
-    return listen(ports[0]);
-  });
-}
+
+cluster(function() {
```

Comment:
Does the function given to `cluster` need to instantiate the app? [Their documentation](https://www.npmjs.com/package/express-cluster#synopsis) seems to indicate so.